Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/phala-dashboards.md

---
id: phala-dashboards
title: Phala Dashboards
sidebar_label: Phala
description:

>   Phala is a parachain on Polkadot that provides a privacy-preserving computation and data
>   protection framework. It allows for the processing of sensitive data with guaranteed privacy.

Reviewer: ```suggestion
  Phala provides a privacy-preserving computation and data
  protection framework. 
```

keywords: [polkadot, dashboard, dune, phala, privacy]
slug: ../phala-dashboards
---

# Phala Dashboards

## Overview

Phala is a parachain on Polkadot that provides a privacy-preserving computation and data protection
framework. It allows for the processing of sensitive data with guaranteed privacy.

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Phala parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Phala parachain:
```

- [Phala Dashboard](https://dune.com/substrate/phala): A comprehensive view of activities within the
  Phala ecosystem.

Please also visit our dashboards for Phala on
[Dune Analytics](https://dune.com/discover/content/relevant?q=title:Phala%20author:substrate).

## Key Tables

Data from the Phala parachain is organized into several key tables:

- `phala.balances`
- `phala.blocks`
- `phala.calls`
- `phala.events`,
- `phala.extrinsics`
- `phala.transfers`

Additional curated dataset from Phala team can be found at `dune.phala_network.*`

## Useful Queries

> Currently, no specific queries have been provided. Please check back later for updates.

Reviewer: ```suggestion
Currently, no specific queries have been provided.
```
